source/sass/global.scss

```diff
@@ -1,5 +1,6 @@
 body > .wrapper {
   background: $white;
+  color: $black;
```

Comment:
yay @taisdesouzalessa 's first PR 🎉 

---

When you inspect the DOM element you see that text color came from CSS rule set on `body`, not `body > .wrapper`

```scss
body {
...
  color: #4d4f53;
...
}
```

<img width="1432" alt="screen shot 2017-04-13 at 3 07 00 pm" src="https://cloud.githubusercontent.com/assets/2896608/25026357/f31559b2-205a-11e7-8d7f-386b6a1263ce.png">

What we wanna do here is to change the color code to `$black`. To do this, we can simply add the following to the top of the file.

```scss
body {
  color: $black;
}
```